tests/unit/test_mysql.py

```diff
@@ -870,10 +873,15 @@ def test_error(self):
         self.assertEqual(error.message, "Error message")
 
     @patch("charms.mysql.v0.mysql.MySQLBase._run_mysqlsh_script")
-    def test_delete_users_for_relation_failure(
-        self,
-        _run_mysqlsh_script,
-    ):
+    def test_delete_users_for_unit(self, _run_mysqlsh_script):
```

Comment:
nit: should we add `_failure` suffix to indicate it is a test for failure?

Reply:
Good catch! I see that suffix is also present in the test right below this one.